packages/runtime/src/templates/server.ts

```diff
@@ -78,12 +77,11 @@ const getNetlifyNextServer = (NextServer: NextServerType) => {
     // doing what they do in https://github.com/vercel/vercel/blob/1663db7ca34d3dd99b57994f801fb30b72fbd2f3/packages/next/src/server-build.ts#L576-L580
     private netlifyPrebundleReact(path: string) {
       const routesManifest = this.getRoutesManifest?.()
-      const appPathsManifest = this.getAppPathsManifest?.()
+      const appPathsRoutes = this.getAppPathRoutes?.()
```

Comment:
Makes sense since `this.getAppPathRoutes()` calls `this.getAppPathsManifest()`.

https://github.com/vercel/next.js/blob/4ddb6fc794201461bea80df5917b6f0a75f62986/packages/next/src/server/base-server.ts#L1042-L1053

Reply:
Yeah exactly, it calls it and then does the normalization I was trying to do myself!